src/quickgui/plugin/qgsquickpositionmarker.qml

```diff
@@ -33,7 +33,12 @@ Item {
   /**
    * Utils for handling position.
    */
-  property QgsQuick.PositionKit positionKit: QgsQuick.PositionKit {id: positionKit}
+  property QgsQuick.PositionKit positionKit: QgsQuick.PositionKit
+  {id: positionKit
+    mapSettings: mapSettings
+    simulatePositionLongLatRad: [-97.36, 36.93, 2]
+    onProjectedPositionChanged: update_location()
```

Comment:
move this outside, use connections" http://doc.qt.io/qt-5/qml-qtqml-connections.html